autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -276,6 +276,12 @@ def fit(train_data,
                             problem_types: (list) List of valid problem types for the model. `problem_types=['binary']` will result in the model only being trained if `problem_type` is 'binary'.
                             disable_in_hpo: (bool) If True, the model will only be trained if `hyperparameter_tune=False`.
                         Reference the default hyperparameters for example usage of these options.
+                    _ag_args_fit: Dictionary of model fit customization options related to AutoGluon. These parameters affect stacker fold models, but not stacker models themselves.
+                        Valid keys:
+                            max_memory_usage_ratio: (float, default=1.0) The ratio of memory usage relative to the default to allow before early stopping or killing the model. Values greater than 1.0 will be increasingly prone to out-of-memory errors.
+                            max_time_limit_ratio: (float, default=1.0) The ratio of given time_limit to use during model `fit()`. If `time_limit=10` and `max_time_limit_ratio=0.3`, time_limit would be changed to 3.
+                            max_time_limit: (float, default=None) Max time_limit value during model `fit()`. If the provided time_limit is greater than this value, it will be replaced by max_time_limit. Occurs after max_time_limit_ratio is applied.
+                            min_time_limit: (float, default=0) Min time_limit value during model `fit()`. If the provided time_limit is less than this value, it will be replaced by min_time_limit. Occurs after max_time_limit is applied.
```

Comment:
"Min time_limit value during model `fit()`. If the provided time_limit is less than this value, it will be replaced by min_time_limit."
->
"Allow this model to train for at least this long (in sec), regardless of the time-limit it would otherwise be granted."

Reply:
Reworded and clarified